chpl/chpl-service/src/main/java/gov/healthit/chpl/surveillance/report/SurveillanceReportManager.java

```diff
@@ -551,31 +551,16 @@ private void reviewQuarterlyReportForDeprecatedFields(QuarterlyReport toCreate)
         if (toCreate.isAcknowledgeWarnings()) {
             return;
         }
-        if (!StringUtils.isEmpty(toCreate.getReactiveSummary()) || !StringUtils.isEmpty(toCreate.getTransparencyDisclosureSummary())) {
-            HashSet<String> errors = new HashSet<String>();
-            HashSet<String> warnings = new HashSet<String>();
-            warnings.add("Deprecated fields will not be used when creating a Quarterly Report");
-            throw new ValidationException(errors, warnings);
-        }
+        //if fields are used that are deprecated create a Set of warnings
+        //and throw a ValidationException with those warnings from here
```

Comment:
Is something missing here?

Reply:
There were no remaining deprecated fields to check against... I thought about removing the method but felt it was a good placeholder for something that will likely happen again in the future. I left a comment to tell that future developer what to do with it. I could be convinced to delete the method if you think that's better.